assets/css/layout/_header.scss

```diff
@@ -0,0 +1,149 @@
+
+        // Toggle menu button on mobile design
+        // Override the default button styles from _base.scss and apply the new style
+        button {
+            background: none;
+            border: none;
+            border-radius: 0;
+            box-shadow: none;
+            color: $white;
+            font-size: rem(16);
+            padding: rem(8);
+        }
+
+        button::before {
+            content: none;
+        }
+        // End of overriding the default button styles from _base.scss
+
+        button:hover {
+            background-color: $orange-focus;
+        }
+
+        // Styles for the navigation menu
+        nav {
+            background: $navy-dark;
+            display: none;
+            left: 0;
+            position: absolute;
+            top: rem(120);
+            width: 100%;
+
+            a {
+                color: $white;
+                display: block;
+                float: none;
+                font-weight: $font-weight-normal;
+                margin: rem(16) rem(16);
+                text-align: left;
+                white-space: nowrap;
+
+                &:hover,
+                &:focus {
+                    color: $navy-light;
+                }
+            }
+        }
+
+        button[aria-expanded="true"] + nav {
+            display: block;
+        }
+
+        // Styles for the search container
+        .search-container {
+            align-items: center;
+            display: flex;
+
+            input[type="search"] {
+                background-color: white;
+                background-image: url("../../static/img/search.svg");
+                background-position: center;
+                background-repeat: no-repeat;
+                border: 0;
+                border-radius: 50%;
+                outline: none;
+                padding: rem(8) rem(16);
+                width: rem(16);
+
+                &:focus {
+                    background-position: left rem(8) center;
+                    border-radius: rem(28);
+                    padding-left: rem(32);
+                    width: rem(160);
+                }
+            }
+        }
+    }
+}
+
+@media screen and (min-width: 600px) {
+    header {
+        .site-nav {
+            justify-content: space-around;
+
+            button {
+                display: none;
+            }
+
+            nav {
+                background-color: transparent;
+                display: flex;
+                flex: 1;
+                flex-direction: row;
+                justify-content: space-around;
+                position: inherit;
+
+                a {
+                    margin: 0;
+                    padding: rem(32) rem(10);
+                }
+
+                // When the user is in a section (other than the Home page), underline this particular item
+                // in the top level navigation.
+                a.nuxt-link-active {
+                    border-bottom: rem(16) solid $orange-focus;
```

Comment:
The border feels a little heavy to me. Not sure if @lliskovoi has any design input.

<img width="774" alt="Screen shot of menu showing current item indicator." src="https://user-images.githubusercontent.com/605361/77354917-33d5bc00-6d09-11ea-8b6b-57663c8da11c.png">